### PR TITLE
added getVisibileColumns method for different utility.

### PR DESCRIPTION
**Pull request by enayet-repo**
added getVisibileColumns method for different utility.

### CODE REVIEW

**Comment by enayet-repo** [comment]
Previous pull request #1943 is closed for dirty branch and commit. New clean PR with documentation edited.

Thanks

**Comment by djhvscf** [comment]
Thanks @enayet-repo for your help

File: docs/_i18n/es/documentation/methods.md [hunk 1]
```diff
@@ -229,6 +229,11 @@ Sintaxis para llamar a un método: `$('#table').bootstrapTable('method', paramet
         <td>Obtiene las columnas ocultas.</td>
     </tr>
     <tr>
+        <td>getVisibleColumns</td>
+        <td>-</td>
+        <td>Recibe las columnas visibles.</td>
```
Comment (djhvscf) [inline docs/_i18n/es/documentation/methods.md]: Here you should use this: **_"Retorna las columnas visibles."_**

File: docs/_i18n/zh-cn/documentation/methods.md [hunk 2]
```diff
@@ -247,6 +247,11 @@
         <td>获取隐藏的列。</td>
     </tr>
     <tr>
+        <td>getVisibleColumns</td>
+        <td>-</td>
+        <td>獲取可見列。</td>
```
Comment (wenzhixin) [inline docs/_i18n/zh-cn/documentation/methods.md]: Here you should use this: "获取可见列。"

**Comment by wenzhixin** [comment]
You can use `git commit --amend` and `git push --force` :)

**Comment by enayet-repo** [comment]
Done. Thanks for suggestion.

**Comment by wenzhixin** [comment]
Thanks!

**Comment by enayet-repo** [comment]
Thanks a lot. I have learned a lot through this at least!!

**Comment by djhvscf** [comment]
@enayet-repo it sounds great!! :+1:

**Pull request merged**